### PR TITLE
 cli: implement commands to open and close interactive sessions

### LINKED ISSUE #29: cli: implement commands to open and close interactive sessions
Create new `open` and `close` commands to work with interactive sessions and replicates all the functionality we have in reana-client.
It should also include all the flags and return exactly the same output as related reana-client commands.

### PR DESCRIPTION
**Pull request by BrunoRosendo**
closes #29

### CODE REVIEW

**Review by mdonadoni — COMMENTED** [review]
Works as expected! There are just a few comments to be addressed. This PR also needs to be rebased.

File: cmd/close.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+			if token == "" {
+				token = os.Getenv("REANA_ACCESS_TOKEN")
+			}
+			workflow, _ := cmd.Flags().GetString("workflow")
+			if workflow == "" {
+				workflow = os.Getenv("REANA_WORKON")
+			}
+
+			validation.ValidateAccessToken(token)
+			validation.ValidateWorkflow(workflow)
```
[the remark this answers is not in the document]
Comment (BrunoRosendo) [reply cmd/close.go:50]: Yes, I don't like the repeated code either. I opened #45

File: cmd/list.go [hunk 2]
```diff
@@ -90,11 +90,8 @@ progress, its duration as of now will be shown.`)
 	return cmd
 }
 
-func list(cmd *cobra.Command, serverURL string) {
-	token, _ := cmd.Flags().GetString("access-token")
-	if token == "" {
-		token = os.Getenv("REANA_ACCESS_TOKEN")
-	}
+func list(cmd *cobra.Command, token string) {
+	serverURL := os.Getenv("REANA_SERVER_URL")
```
[the remark this answers is not in the document]
Comment (BrunoRosendo) [reply cmd/list.go:94]: Sure, I can change every command to pass env vars as a parameter as well. We should try to keep everything consistent

**Review by mdonadoni — APPROVED** [review]
LGTM! This PR needs to be rebased before being merged

**Pull request merged**